stories/Button/Button.stories.tsx

```diff
@@ -1,13 +1,17 @@
-/* eslint-disable react/prop-types */
+// @ts-ignore
```

Comment:
Rather than `@ts-ignore` can you use `@ts-expect-error` with a description of the reason why it was added?

Reply:
reverted `eslint-disable react/prop-types`